modules/ossm-mt-namespaces.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-mt-ossm.adoc
+
+[id="ossm-mt-namespaces_{context}"]
+= Configuring namespaces in multi-tenant installations
+
+A multi-tenant control plane installation only affects namespaces configured as part of the {ProductShortName}. You must specify the namespaces associated with the {ProductShortName} in a `ServiceMeshMemberRoll` resource located in the same namespace as the `ServiceMeshControlPlane` resource and name it `default`. Here is an example that joins the bookinfo namespace into the {ProductShortName}:
+
+.Prerequisites
+
+* An installed, verified {ProductShortName} operator.
+* Location of the namespace where the custom resource used to install the control plane.
```

Comment:
Location of the namespace where where the custom resource installed the control plane. No need for used here.

Reply:
Fixed.